test/usersignup.go

```diff
@@ -62,9 +63,19 @@ func DeactivatedWithLastTransitionTime(before time.Duration) UserSignupModifier
 	}
 }
 
-func VerificationRequired() UserSignupModifier {
+func VerificationRequired(before time.Duration) UserSignupModifier {
 	return func(userSignup *toolchainv1alpha1.UserSignup) {
 		states.SetVerificationRequired(userSignup, true)
+
+		deactivatedCondition := toolchainv1alpha1.Condition{
```

Comment:
so all users with `VerificationRequired` will get an extra `deactivatedCondition`? what about the tests for fresh/first-time users?

Reply:
I copy-pasted the part and forgot to rename the condition var: bbb58c2

Reply:
ok, thanks, I got confused with the condition name 🤦‍♂️